src/app/modules/main/communities/module.nim

```diff
@@ -182,45 +179,22 @@ method curatedCommunitiesLoadingFailed*(self: Module) =
   self.curatedCommunitiesLoaded = true
   self.view.setCuratedCommunitiesLoading(false)
 
-proc createMemberItem(self: Module, memberId, requestId: string, status: MembershipRequestState): MemberItem =
-  let contactDetails = self.controller.getContactDetails(memberId)
```

Comment:
I don't get this part; how do the member roles/props get initialized now? You could've kept most of the code intact imo, and just init the expensive stuff to empty